skins/laika/src/components/pages/UpdateAddress.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+	<div class="address-update-form">
+		<form name="address-update" ref="form" :action="updateAddressURL" method="post">
+			<h1 class="title is-size-1">{{ $t('address_change_form_title') }}</h1>
```

Comment:
there's no space around `'address_change_form_title'` and I know the linter doesn't complain, it is unfortunate.
```suggestion
			<h1 class="title is-size-1">{{ $t( 'address_change_form_title' ) }}</h1>
```
I was trying to change the eslint configs to do that but had no luck so far.